api/api-rules/violation_exceptions.list

```diff
@@ -161,6 +161,7 @@ API rule violation: list_type_missing,k8s.io/api/core/v1,ServiceSpec,ExternalIPs
 API rule violation: list_type_missing,k8s.io/api/core/v1,ServiceSpec,LoadBalancerSourceRanges
 API rule violation: list_type_missing,k8s.io/api/core/v1,TopologySelectorLabelRequirement,Values
 API rule violation: list_type_missing,k8s.io/api/core/v1,TopologySelectorTerm,MatchLabelExpressions
+API rule violation: list_type_missing,k8s.io/api/core/v1,TopologySpreadConstraint,MatchLabelKeys
```

Comment:
Please define a list type for the new field, to avoid adding new lint exceptions. If you expect a single actor to control the whole list, then `+listType=atomic` seems most appropriate for this field.

Reply:
updated